components/secure_embed/browser/secure_embed_browsertest.cc

```diff
@@ -423,4 +423,45 @@ IN_PROC_BROWSER_TEST_F(SecureEmbedBrowserTest, ReattachSameGuestToNewEmbed) {
   VerifyBoxRendering(SK_ColorRED);
 }
 
+#if defined(USE_AURA)
+// Test that the proper (top-level) InputManager is used for embedded
+// WebContents. BoundingBoxUpdateWaiter is aura-specific. This triggers
+// a selection change on top-level when the embed is focused.
+IN_PROC_BROWSER_TEST_F(SecureEmbedBrowserTest, InputManager) {
+  auto guest_contents =
+      SetupHarnessAndGuestWithContent("/secure_embed/red_box.html");
+  AttachGuestToEmbed(guest_contents.get());
+
+  content::BoundingBoxUpdateWaiter waiter(web_contents());
```

Comment:
Can we move this line above L456, which is closer to where the waiter is used? If not, can we document the reason?